python/lsst/afw/cameraGeom/utils.py

```diff
@@ -419,17 +504,27 @@ def getCcdImage(self, ccd, imageFactory=afwImage.ImageF, binSize=1):
 
 def rawCallback(im, ccd=None, imageSource=None,
                 correctGain=False, subtractBias=False, convertToFloat=False):
-    """!A callback function that may or may not subtract bias/correct gain/trim a raw image
-
-    @param im   An image of a chip, ready to be binned and maybe rotated.  Maybe an Exposure,
-                a MaskedImage or an Image
-    @param ccd  The Detector; if None assume that im is an exposure and extract it's Detector
-    @param imageSource  An object derived from FakeImageDataSource (typically ButlerImage)
-    @param correctGain  Correct each amplifier for its gain
-    @param subtractBias Subtract the bias from each amplifier
-    @param convertToFloat  Convert im to floating point if possible
-
-    N.b. if imageSource is derived from ButlerImage, imageSource.butler is available
+    """A callback function that may or may not subtract bias/correct gain/trim
+    a raw image.
+
+    Paramters
+    ---------
+    im : `lsst.afw.image.Image` or `lsst.afw.image.MaskedImage` or `lsst.afw.image.Exposure`
+       An image of a chip, ready to be binned and maybe rotated.
+    ccd : `lsst.afw.cameraGeom.Detector` or None
+        The Detector; if None assume that im is an exposure and extract its Detector.
+    imageSource : `lsst.afw.image.Image` or None
+        An object derived from FakeImageDataSource (typically ButlerImage).
+    correctGain : `bool`
+        Correct each amplifier for its gain.
+    subtractBias : `bool`
+        Subtract the bias from each amplifier.
+    convertToFloat : `bool`
+        Convert im to floating point if possible.
+
+    Notes
+    -----
+    If imageSource is derived from ButlerImage, imageSource.butler is available.
```

Comment:
Returns?

Reply:
Even if we're updating the formatting I don't think we should be putting in guesses.

Reply:
That's what I mean by "guess": I looked at the source and made an educated guess:

```
    Returns
    -------
    image : `lsst.afw.image.Image` like
        The constructed image (type returned by ``im.Factory``).
```

However, you're the author here, so what do you suggest?